packages/components/pages/f-loyalty/src/components/Slots.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+    <div>
+        <div
+            v-if="!hasInProgressStampcards && !hasRedeemableStampcards"
+        >
```

Comment:
Please can this be collapsed:
```suggestion
        <div v-if="!hasInProgressStampcards && !hasRedeemableStampcards">
```